src/app.js

```diff
@@ -12,6 +12,12 @@ app.use((req, res, next) => {
   logger.debug(`${req.method} ${req.url}`);
   next();
 });
+app.use((req, res, next) => {
```

Comment:
Another option is to use https://github.com/expressjs/cors

Reply:
Thanks. I switched to that